src/main/resources/META-INF/definitions.cnd

```diff
@@ -167,9 +167,6 @@
  - toAdminComponent (string, choicelist[resourceBundle]) = 'all' indexed=no < 'all', 'users', 'groups'
  - buttonClass (string) indexed=no
 
-[jnt:gotoManager] > jnt:content, mix:title, jmix:siteComponent, jmix:studioOnly
```

Comment:
Now you removed the whole component - I don't know if it's a good idea, I don't know where it was used -  but in that case you should remove the jsp .. ?

Reply:
I'll close this PR and start from the beginning